src/sysusers/sysusers.c

```diff
@@ -1346,9 +1346,11 @@ static int process_item(Item *i) {
         switch (i->type) {
 
         case ADD_USER: {
-                Item *j;
+                Item *j = NULL;
+
+                if (!(i->gid_set))
```

Comment:
redundant set of ()

Reply:
Indeed, operator precedence is enough for this. Reworked and dropped the extra parenthesis.